fsw/src/sample_table.c

```diff
@@ -1,6 +1,6 @@
 /*
 **
-**      GSC-18128-1, "Core Flight Executive Version 6.6"
```

Comment:
Look into this change as discussed in CCB 20200318

Reply:
Confirmed that this is wrong/incorrect in master branch, it was just inadvertently "fixed" here when I did a fix for the other (new rebased) files in this pull.

At any rate, the change _should_ be applied - the question is whether it is OK to leave it here in this pull or if we need to withdraw it from here and make a separate issue/pull to fix the comment in the existing file.

My preference would be to just merge this and fix it/move on, unless the process experts say otherwise....

Reply:
Now I'm curious what I did wrong that this slipped through.... anyways, no need for a separation.  Just fix and move on.

Reply:
Oh.. grepped the entire repo and this table was the only issue I found.  I bet there was a merge when the table was added that we didn't catch the version mismatch.

Reply:
Edit - entire bundle